Packs/FeedTorExitAddresses/Integrations/FeedTorExitAddresses/FeedTorExitAddresses.py

```diff
@@ -0,0 +1,127 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+import requests
+from dateutil.parser import parse
+from typing import Dict, List
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+SOURCE_NAME = "Tor Exit Addresses"
+
+
+class Client(BaseClient):
+    def __init__(self, insecure: bool = False, proxy: bool = False):
+        super().__init__(base_url='https://check.torproject.org/exit-addresses', verify=not insecure, proxy=proxy)
+        self.url = 'https://check.torproject.org/exit-addresses'
+
+    def http_request_indicators(self):
+        res = requests.get(
+            url=self.url,
+            verify=self._verify
+        )
+
+        try:
+            res.raise_for_status()
+
+        except Exception:
+            LOG(f'Tor Exit Addresses - exception in request: {res.status_code!r} {res.content!r}')
+            raise
+
+        return res.text
+
+    def datestring_to_millisecond_timestamp(self, date_string):
+        date = parse(str(date_string))
+        return int(date.timestamp() * 1000)
+
+    def build_iterator(self, limit):
+        raw_res = self.http_request_indicators()
+        raw_indicator_list = raw_res.split('\n')
+        indicator_list = []  # type: List
+        indicator = {}  # type: Dict
+        current_indicator_index = 0
+        for line in raw_indicator_list:
+            if line.startswith('ExitNode'):
+                indicator = {
+                    "type": FeedIndicatorType.IP,
+                    'name': line.split()[1]
+                }
+
+            elif line.startswith('Published'):
+                date = line.split(' ', 1)[1]
+                indicator['firstseenbyfeed'] = self.datestring_to_millisecond_timestamp(date)
+
+            elif line.startswith('LastStatus'):
+                date = line.split(' ', 1)[1]
+                indicator['lastseenbyfeed'] = self.datestring_to_millisecond_timestamp(date)
+
+            elif line.startswith('ExitAddress'):
+                indicator['value'] = line.split()[1]
+                raw_json = indicator.copy()
+                indicator['rawjson'] = raw_json
+                indicator['fields'] = {
+                    'firstseenbyfeed': indicator.get('firstseenbyfeed'),
+                    'lastseenbyfeed': indicator.get('lastseenbyfeed'),
+                    'name': indicator.get('name')
+                }
+                indicator_list.append(indicator)
```

Comment:
we should run the for on 4 lines at a time, this seems weird this way

Reply:
well... could be kind of complicated since apparently from time to time the structure kind of changes - I'll show you in person